modules/guac/ra_guac_autoscale_public_alb.template.cfn.yaml

```
@@ -668,6 +753,11 @@ Resources:
               mode: '000700'
               owner: root
               source: !Sub ${RemoteAccessScriptsUrl}/scripts/make-guac.sh
+            /usr/local/tomcat/conf/server.xml:
```

It feels odd to put this in the "real" location when the host is not running tomcat... maybe use the "staging" location for cfn files?

```suggestion
            /etc/cfn/files/tomcat/conf/server.xml:
```

---

Okay.  I'll use your suggestion.